docs/source/archive/gsoc/reports/2024/vulnerablecode_ambuj.rst

```diff
@@ -0,0 +1,175 @@
+======================================================================================
+Add more data sources and mine the graph to find correlations between vulnerabilities
```

Comment:
This title was fine for the project idea (this is a running project for several years, title hasn't been updated), but we should modify this to suit the project more.
Maybe  `Add more datasources and improve vulneribility data`?
@ziadhany @TG1999 what do you think?

Reply:
@AyanSinhaMahapatra  Add more data sources and improve data quality 
 this is the name of my project board should I use it as a title.

Reply:
Yeah, sounds good.